site/en/Test.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+lang-ref: first test
+lang: en
+---
+
+# Results and findings of the first OCR-D test
+## Background
+At the turn of the year 2019/2020, the OCR-D software was tested for the first time in nine pilot libraries. This was to ensure the practical acceptance of the software by future, potential users. Therefore the focus was on its functionality and usability in practice. In addition to the partners of the coordination project, two libraries involved in the module projects as well as four other libraries took part in the testing. The findings of this first test run will be incorporated into the further development of the OCR-D prototype.
+
+All pilot libraries have some know-how and experience with OCR, as they have already produced full texts at least at project level or via service providers. The extent to which OCR, which is regarded as important, should be carried out independently in future and firmly anchored in the digitisation workflow, is currently discussed in the libraries. Concerning the target groups of the full texts there are different opinions in the testing institutions. While one third generally name humanities scholars, another third would like to serve a very broad target group (humanities, digital humanities, computer linguistics and economics). The remaining libraries only see relatively few, specialised users (Digital Humanities or Computational Linguistics) as the target group for OCR texts.
```

Comment:
at project level -> on project level

service providers ->? commercial service providers/consultants

independently -> in-house

generally name -> name humanities (scholars) in general